### PR TITLE
Add web bridge migration guides for iOS and Android v5

### PR DESCRIPTION
**Pull request by Sporiff**
Closes https://adjustcom.atlassian.net/browse/ADAPP-18258

### CODE REVIEW

File: src/content/docs/sdk/android/migration/web-v4-to-v5.mdx [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+Here's what you need to do before updating to SDK v5:
+
+1. SDK v5 supports [SDK signature verification](https://help.adjust.com/en/article/sdk-signature) natively. If you currently use the SDK signature library, you need to uninstall this first.
+2. If your app targets API versions lower than [**21**](https://developer.android.com/tools/releases/platforms#5.0) you need to update your app before you can use SDK v5.
+
+## [Install the beta SDK](install-the-sdk)
+
+To start using SDK v5, you need to add it as a dependency in your project. To do this:
+
+1. Add the beta repository to your `build.gradle` file.
```
Comment (shashanksu) [inline src/content/docs/sdk/android/migration/web-v4-to-v5.mdx:26]: This is only applicable for beta.  With v5 stable release, this step would not be needed.

File: src/content/docs/sdk/android/migration/web-v4-to-v5.mdx [hunk 2]
```diff
@@ -0,0 +1,591 @@
...
+   </CodeBlock>
+
+2. Remove any older versions of the SDK you currently have in your `build.gradle` file.
+3. Add the following dependencies to your `build.gradle` file:
+
+   <CodeBlock title="build.gradle">
+
+   ```groovy
+   dependencies {
+      implementation 'com.adjust.sdk:adjust-android:5.0.0-SNAPSHOT'
```
Comment (shashanksu) [inline src/content/docs/sdk/android/migration/web-v4-to-v5.mdx:47]: This is only applicable for beta. With v5 stable release the version would be 5.0.0 (without `-SNAPSHOT`)

File: src/content/docs/sdk/android/migration/web-v4-to-v5.mdx [hunk 3]
```diff
@@ -0,0 +1,591 @@
...
+</MinorVersion>
+
+```js
+adjustConfig.setCoppaCompliantEnabled(true);
+```
+
+In SDK v5, this method is renamed to `enableCoppaCompliance` for clarity. This new method accepts no arguments.
+
+```js
+Adjust.enableCoppaCompliance();
```
Comment (shashanksu) [inline src/content/docs/sdk/android/migration/web-v4-to-v5.mdx:376]: This is now again changed to:
```
adjustConfig.enableCoppaCompliance();
```

File: src/content/docs/sdk/android/migration/web-v4-to-v5.mdx [hunk 4]
```diff
@@ -0,0 +1,591 @@
...
+</MinorVersion>
+
+```js
+adjustConfig.setPlayStoreKidsAppEnabled(true);
+```
+
+In SDK v5, this method is renamed to `enablePlayStoreKidsCompliance` for clarity. This new method accepts no arguments.
+
+```js
+Adjust.enablePlayStoreKidsCompliance();
```
Comment (shashanksu) [inline src/content/docs/sdk/android/migration/web-v4-to-v5.mdx:394]: This is now again changed to:
```
adjustConfig.enablePlayStoreKidsCompliance();
```

**Review by shashanksu — CHANGES_REQUESTED** [review]

**Comment by Aditi3** [comment]
Hi @Sporiff 

For iOS Web Bridge migration guide the [following section](https://dev.adjust.com/en/sdk/ios?version=v4#integrate-adjustbridge-into-your-web-view) has been removed and updated with below code snippet - 

```
 var yourAppToken = yourAppToken;
 var environment = AdjustConfig.EnvironmentSandbox;
 var adjustConfig = new AdjustConfig(yourAppToken, environment);
 Adjust.initSdk(adjustConfig);
```

**Comment by Aditi3** [comment]
For FB Pixel, 

The code snippet over here - https://dev.adjust.com/en/sdk/ios/plugins/facebook-pixel/#adjust-sdk-integration, changes to -

```
- (void)viewWillAppear:(BOOL)animated {
   
   WKWebView *wkWebView = [[WKWebView alloc] initWithFrame:self.view.frame];
    /// add @property (nonatomic, strong) AdjustBridge *adjustBridge; on your interface
   self.adjustBridge = [[AdjustBridge alloc] init];
   [self.adjustBridge augmentHybridWebView];
   [self.adjustBridge loadWKWebViewBridge:wkWebView];
    
  }
```

**Comment by Sporiff** [comment]
@uerceg @Aditi3 The iOS web bridge migration guide is available [here](https://dev-docs-git-adapp-18258-add-web-bridge-1864b7-product-content.vercel.app/en/sdk/ios/migration/web-v4-to-v5)

I will be adding the JS code snippets to the relevant places in the documentation for the next little while. I'll ping again when it's done. After that, I'll merge everything and update #948 so that you can take a look at how everything is going to look prior to merge.

**Comment by uerceg** [comment]
@Aditi3 Can double check the code syntax, but yeah, the initialization snippets still to be added and I believe that the part for adding web bridge via CocoaPods also need to be changed (@Aditi3 can confirm that part as well).

File: src/content/docs/sdk/ios/migration/web-v4-to-v5.mdx [hunk 5]
```diff
@@ -0,0 +1,666 @@
...
+
+To start using SDK v5, you need to add it as a dependency in your XCode project. To do this:
+
+1. Remove any older versions of the SDK you currently have in your `Podfile`.
+2. Add the following line to your `Podfile`:
+
+   <CodeBlock title="Podfile">
+
+   ```rb
+   pod 'Adjust', :git => 'https://github.com/adjust/ios_sdk.git', :branch => 'v5'
```
Comment (Aditi3) [inline src/content/docs/sdk/ios/migration/web-v4-to-v5.mdx:28]:
```suggestion
   pod 'Adjust/AdjustWebBridge', :git => 'https://github.com/adjust/ios_sdk.git', :branch => 'v5'
```

Comment (uerceg) [reply src/content/docs/sdk/ios/migration/web-v4-to-v5.mdx:28]: Is this aiming beta or official release? If it's for the official release, then:

```
pod 'Adjust/AdjustWebBridge', :git => 'https://github.com/adjust/ios_sdk.git', :tag => 'v5.0.0'
```

**Comment by Aditi3** [comment]
> @Aditi3 Can double check the code syntax, but yeah, the initialization snippets still to be added and I believe that the part for adding web bridge via CocoaPods also need to be changed (@Aditi3 can confirm that part as well).

Yes, this could be part of migration guide - https://github.com/adjust/dev-docs/pull/947#issuecomment-2263284222

**Comment by Sporiff** [comment]
@uerceg @Aditi3 @shashanksu I'm merging this branch into #948 so you can go through and review everything in one place. I'll ping you the Android/iOS link once the build is finished.

**Pull request merged**